<!-- -*- xml -*-

     This tutorial relies on a live video sequence acquired with an 
     usb camera in front of a QR code planar target. The model 
     corresponding to this target is given into the models directory 
     of this package.

     Camera parameters are set as rosparam parameters. They need to be changed
     to be the one of your camera.

     See http://www.ros.org/wiki/visp_auto_tracker for more information.

  -->
<launch>	
  <!-- Launch the tracking node -->
  <node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker" output="screen">
    <param name="model_path" value="$(find walle)/src/models" />
    <param name="model_name" value="qrDemoDay" />
    <param name="debug_display" value="True" />
	      
    <remap from="/visp_auto_tracker/camera_info" to="/usb_cam/camera_info"/>
    <remap from="/visp_auto_tracker/image_raw" to="/usb_cam/image_raw/compressed"/>
  </node>

visp_auto_tracker/models/calibration.ini file --> 
    <param name="camera_name" value="/camera/image_raw/compressed" />
    <param name="camera_info_url" value="package://walle/src/models/calibrationAstra.ini" type="string" />

<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
    <param name="image_transport" value="compressed" />
  </node>

</launch>
